sdk/include/opentelemetry/sdk/metrics/metric_reader.h

```diff
@@ -9,69 +9,55 @@
 #  include "opentelemetry/sdk/metrics/instruments.h"
 #  include "opentelemetry/version.h"
 
+#  include <memory>
+
 OPENTELEMETRY_BEGIN_NAMESPACE
 namespace sdk
 {
 namespace metrics
 {
 
+class MetricProducer;
 /**
  * MetricReader defines the interface to collect metrics from SDK
  */
 class MetricReader
 {
 public:
   MetricReader(
-      AggregationTemporarily aggregation_temporarily = AggregationTemporarily::kCummulative)
-      : aggregation_temporarily_(aggregation_temporarily), measurement_processor_callback_({})
-  {}
+      AggregationTemporarily aggregation_temporarily = AggregationTemporarily::kCummulative);
 
```

Comment:
virtual dtor is removed.

Reply:
Thanks fixed.

Reply:
Not sure if it's my browser cache. I see the dtor is is still removed.

Reply:
I can see in the PR: https://github.com/lalitb/opentelemetry-cpp/blob/7bc1142cfd0374d789f07f2cbdc423968e41505b/sdk/include/opentelemetry/sdk/metrics/metric_reader.h#L52

I moved it from top to bottom, maybe that is causing confusion :)